docs/quick_start.md

```diff
@@ -1,12 +1,34 @@
 # Quick Start
 
 The purpose of this quick start is to get you up and running with the {{open_autonomy}} framework for agent service development as quickly as possible.
+The overall pipeline with the framework is summarized in the figure below:
+
+<figure markdown>
+![](./images/pipeline.svg)
+<figcaption>Overall pipeline to deploy an agent service with the Open Autonomy framework</figcaption>
+</figure>
+
+The steps defined in the figure above are materialized as follows:
+
+1. **Component provision:** The developer ensures that the agent components are available, either by programming them from scratch using the {{open_aea}} framework, or fetching them from the Component Registry. Each agent service needs one or more underlying {{fsm_app}}(s) that define the service business logic. The {{open_autonomy}} framework contains packages with base classes that help define {{fsm_app}}s.
+
+2. **Agent definition:** The developer ensures that the agents that make up the service are available, either by defining them  using the {{open_aea}} framework, or by fetching them from the Agent Registry.
```

Comment:
I think there are two different parts here: first, the developer defines the agent (its components, configuration...). Then the service is defined. After that, operators register their instances.